Stubs update move to branch

billy-the-fish opened the pull request:
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

ujmediaservices committed:
Merging up stubs into main documents, editing, formatting

ujmediaservices committed:
Oops typo

jamessewell committed:
Merge branch 'latest' into stub-merge-1

billy-the-fish committed:
Update for linter

Signed-off-by: Iain Cox <iain.cox@sarkdocumentation.com>

billy-the-fish committed:
Update administration.md

Signed-off-by: Iain Cox <iain.cox@sarkdocumentation.com>

billy-the-fish committed:
Merge branch 'latest' into stub-merge-1

billy-the-fish committed:
fix: update navigation and links for page merge.

billy-the-fish committed:
fix: update navigation and links for page merge.

billy-the-fish committed:
Merge branch 'latest' into stub-merge-1

billy-the-fish committed:
fix: clean up de-chunking.

billy-the-fish committed:
fix: clean up de-chunking.

billy-the-fish committed:
fix: clean up de-chunking.

billy-the-fish committed:
Merge branch 'latest' into stubs-update-move-to-branch

billy-the-fish committed:
Merge branch 'latest' into stubs-update-move-to-branch

billy-the-fish committed:
Merge branch 'latest' into stubs-update-move-to-branch

billy-the-fish committed:
feat: unite stubs.

billy-the-fish committed:
Merge branch 'latest' into stubs-update-move-to-branch

billy-the-fish committed:
Merge branch 'latest' into stubs-update-move-to-branch

billy-the-fish committed:
feat: unite stubs.

billy-the-fish committed:
Merge branch 'latest' into stubs-update-move-to-branch

Commit:
Fix typo (#3052)

Signed-off-by: Florian Mt <fmonfort@enssat.fr>
Co-authored-by: Iain Cox <iain.cox@sarkdocumentation.com>

contentana commented:
I'm sure you're on it, @billy-the-fish, but let's ensure we correctly redirect all the old links to the new ones so we don't break anything. Thanks!

antekresic committed:
Fix typo in troubleshooting section (#3073)

JamesGuthrie committed:
Update live-migration to v0.0.9 (#3071)

Co-authored-by: Iain Cox <iain.cox@sarkdocumentation.com>

alai97 committed:
Copy Nit (#3078)

Signed-off-by: Austin Lai <76412946+alai97@users.noreply.github.com>

solugebefola committed:
Update import-csv.md (#3080)

Make time column references consistent

Signed-off-by: Solar Olugebefola <12288273+solugebefola@users.noreply.github.com>

antekresic committed:
Update new about compression (#3012)

arajkumar committed:
live-migration: Add docs changes for v0.0.10 (#3091)

Signed-off-by: Arunprasad Rajkumar <ar.arunprasad@gmail.com>

minkimipt committed:
Add Google CA (#3090)

We've added a new CA provider to our cloud, reflecting this fact in the
docs.

solugebefola committed:
Update ruby.md (#3065)

* Update ruby.md 

Modify references to Timescale/TimescaleDB/Timescale Cloud

Signed-off-by: Solar Olugebefola <12288273+solugebefola@users.noreply.github.com>

* fix: update TOC links to match headings.

* fix: update TOC links to match headings.

* fix: correct product name.

---------

Signed-off-by: Solar Olugebefola <12288273+solugebefola@users.noreply.github.com>
Co-authored-by: Iain Cox <iain.cox@sarkdocumentation.com>

arajkumar committed:
live-migration: Add docs changes for v0.0.11 (#3096)

Signed-off-by: Arunprasad Rajkumar <ar.arunprasad@gmail.com>

giokostis committed:
Move multinode migration under playbooks (#3110)

billy-the-fish committed:
3113 docs rfc add new editor (#3115)

* First draft

* Complete article, add images

* Add a screenshot of PopSQL

* Update popsql_button_in_console

Highlighted the button

* Wrap text

* feat: update PopSQL page, get started and update sections with PopSQL information.

* feat: link to images on S3.

* fix: language cleanup.

* Apply suggestions from code review

Co-authored-by: sremertimescale <165809978+sremertimescale@users.noreply.github.com>
Signed-off-by: Iain Cox <iain.cox@sarkdocumentation.com>

* fix: language cleanup after review.

* fix: another language review.

* fix: updates following review by Ramon.

* Update use-timescale/popsql.md

Co-authored-by: Ramon Guiu <ramon@timescale.com>
Signed-off-by: Iain Cox <iain.cox@sarkdocumentation.com>

* Update use-timescale/query-data/about-query-data.md

Co-authored-by: Ramon Guiu <ramon@timescale.com>
Signed-off-by: Iain Cox <iain.cox@sarkdocumentation.com>

* fix: add contact information to add more seats.

---------

Signed-off-by: Iain Cox <iain.cox@sarkdocumentation.com>
Co-authored-by: Rahil Sondhi <rahilsondhi@gmail.com>
Co-authored-by: sremertimescale <165809978+sremertimescale@users.noreply.github.com>
Co-authored-by: Ramon Guiu <ramon@timescale.com>

billy-the-fish committed:
fix: numbering and full stop. (#3120)

sdebruyn committed:
missing word in add_reorder_policy (#3106)

Signed-off-by: Sam Debruyn <sam@debruyn.dev>
Co-authored-by: Iain Cox <iain.cox@sarkdocumentation.com>

jnidzwetzki committed:
Restrict usage of time_bucket_ng in CAggs (#3112)

The function time_bucket_ng is deprecated and should not be used in new
CAggs. 2.15.0 will also block the usage of this function.

billy-the-fish committed:
fix: broken links. (#3123)

* fix: broken links.

* fix: broken links.

jnidzwetzki committed:
Update CAgg add_continuous_aggregate_policy documentation (#3128)

billy-the-fish committed:
fix: deprecate distributed hypertables from the API reference. (#3134)

billy-the-fish committed:
fix: update API reference for set_integer_now_func. (#3135)

* fix: update API reference for set_integer_now_func.

Co-authored-by: Jônatas Davi Paganini <jonatas@timescale.com>
Signed-off-by: Iain Cox <iain@timescale.com>

antekresic committed:
Add compression settings information views (#3142)

billy-the-fish committed:
fix: using the public API to check the  chunk time interval for a hypertable. (#3141)

billy-the-fish committed:
3144 docs rfc be more clear about tiered storage being on timescale not on self hosted (#3145)

* feat: update tiered storage overview

* feat: update tiered storage overview

fabriziomello committed:
Update alter_job.md (#3103)

Added missing information in alter_job options

Signed-off-by: Fabrízio de Royes Mello <fabriziomello@gmail.com>
Co-authored-by: Iain Cox <iain@timescale.com>

billy-the-fish committed:
fix: some broken links. A couple of text updates. (#3151)

harkishen committed:
feat: FAQ for live migration (#3139)

Signed-off-by: Harkishen-Singh <harkishensingh@hotmail.com>

harkishen committed:
chore: update live-migration to v0.0.13 (#3150)

Signed-off-by: Harkishen-Singh <harkishensingh@hotmail.com>

JamesGuthrie committed:
Add pgstattuple to list of available extensions (#3154)

billy-the-fish committed:
fix: clarify that each service hosts a single service only. (#3161)

billy-the-fish committed:
fix: change link in Migrate the entire database at once to the live migration page. (#3159)

JamesGuthrie committed:
Add warning about SERIAL types and dual-write (#3162)

jgpruitt committed:
Address possible installation issues regarding psycopg2 (#3165)

* Address possible installation issues regarding psycopg2

* Address possible installation issues regarding psycopg2

* chore: few language updates for the install procedure.

---------

Co-authored-by: billy-the-fish <iain.cox@sarkdocumentation.com>

JamesGuthrie committed:
Add troubleshooting note about partial continuous aggregates (#3164)


Signed-off-by: James Guthrie <JamesGuthrie@users.noreply.github.com>
Co-authored-by: billy-the-fish <iain.cox@sarkdocumentation.com>

billy-the-fish committed:
2829 obsolete restriction on replication factor 1 still in the docs 1 (#3129)

* fix: update create_distributed_hypertable

alejandrodnm committed:
Recommend live-migration instead of dual-writes (#3173)

arajkumar committed:
Refer to live-migration 0.0.14 (#3179)

Signed-off-by: Arunprasad Rajkumar <ar.arunprasad@gmail.com>

harkishen committed:
feat: add version downgrade for live-migration FAQ (#3163)

Signed-off-by: Harkishen-Singh <harkishensingh@hotmail.com>

This commit adds a FAQ question that helps users to downgrade
TimescaleDB version on their target database to match with the
source database. This is useful when the source database (MST) does not
have the TimescaleDB version that the target database is installed
with.

RobAtticus committed:
Make some changes to refresh cagg note (#3146)

* Make some changes to refresh cagg note

This fixes one typo and changes a few other words.

Signed-off-by: RobAtticus <rob@timescale.com>

* chore: updates after review.

---------

Signed-off-by: RobAtticus <rob@timescale.com>
Co-authored-by: Iain <iain@timescale.com>

minkimipt committed:
Add role based auth to CloudWatch exporter doc (#3156)

* Add role based auth to CloudWatch exporter doc

Role based authentication is added for CloudWatch exporter that allows
to authenticate with users' AWS accounts and we need to document
configuration that needs to be done on the user side to make it work.

* add provider URLs and role example

* Clarify what the URL serves for

* chore: first go updating the configuration procedure.

* chore: updates after first review.

* chore: updates after first review.

* Update use-timescale/metrics-logging/integrations.md

Nice.

Co-authored-by: Danil Zhigalin <danil@timescale.com>
Signed-off-by: Iain Cox <iain@timescale.com>

* fix: update metatags for MDX.

* Replace trust policy to a role

AWS docs refer to this as a role. Make sure we are talking about the
same thing in our docs.

* Update use-timescale/metrics-logging/integrations.md

Co-authored-by: Nicolas Nosenzo <niconosenzo@gmail.com>
Signed-off-by: Iain Cox <iain@timescale.com>

---------

Signed-off-by: Iain Cox <iain@timescale.com>
Co-authored-by: niconosenzo <niconosenzo@gmail.com>
Co-authored-by: billy-the-fish <iain.cox@sarkdocumentation.com>
Co-authored-by: Iain Cox <iain@timescale.com>

dtext committed:
fix: document actual behavior of add_continuous_aggregate_policy when using if_not_exists (#3037)

Signed-off-by: dtext <textores.danny@gmail.com>
Co-authored-by: Iain Cox <iain@timescale.com>

billy-the-fish committed:
fix: add verb to sentence. (#3131)

* fix: add verb to sentence.
* fix: update after review.

billy-the-fish committed:
chore: migrate changes in future versions back to this PR.

contentana commented on _partials/_cloud-installation.md:
```diff
@@ -2,14 +2,16 @@
 ### Create a Timescale account
 
-If you have not already signed up for Timescale, join up now:
+To setup Timescale:
```
To set up Timescale:

billy-the-fish committed:
Merge remote-tracking branch 'origin/latest' into stubs-update-move-to-branch

# Conflicts:
#	about/release-notes/index.md
#	api/create_distributed_hypertable.md
#	migrate/page-index/page-index.js
#	migrate/pg-dump-and-restore/pg-dump-restore-from-postgres.md
#	migrate/pg-dump-and-restore/pg-dump-restore-from-timescaledb.md
#	use-timescale/metrics-logging/integrations.md

billy-the-fish committed:
fix: my fatal weakness.

billy-the-fish committed:
fix: one broken link fixed.

billy-the-fish committed:
fix: more link fixing after manual check.
